botfront/imports/lib/story.utils.js

```diff
@@ -363,3 +363,48 @@ export const accumulateExceptions = (
     });
     return exceptions;
 };
+
+export const getStoryEvents = (md, excludeEvents) => {
```

Comment:
Are you passing excludedEvents to avoid duplicates? If yes just use a `Set`

Reply:
yes it's just to avoid duplicates, I will change this right away.

Reply:
Just following up. has this been resolved?

Reply:
this has now been resolved